### ISSUE TITLE
[New Tracker REQ] Waffles

### ISSUE THREAD

**Issue by RoloSoze** [Jackett/Jackett#1182]
**Indexer Name**
Waffles

**Type**
Audio (FLAC, MP3), e-books, audiobooks, apps, e-learning videos 

**Indexer URL**
https://waffles.ch/

**Indexer Language**
English

**Invite**
It would be great if someone is willing to send an invite to a developer

**Note**
Waffles is a large music tracker with around 290K torrents. It had been offline for months but it's back online now.

**Comment by adamwinn** [comment]
I'll take a stab at this tomorrow.  I was already thinking of doing it anyway.

**Comment by RoloSoze** [comment]
@adamwinn Thank you very much! I'll keep the 'invite needed' label for now until you let me know that everything is going well :) Cheers!

**Comment by adamwinn** [comment]
@RoloSoze Can you help me get the rows returned from Waffles? I keep getting "Info Manual search for "rock" on Waffles with 0 results." (I'm searching for "rock")

```
search:
    path: browse.php
    inputs:
      $raw: "{{range .Categories}}filter_cat[{{.}}]=1&{{end}}"
      q: "{{ .Query.Keywords }}"
    rows:
      selector: table#browsetable > tbody > tr:has(a[href^="details.php?id="])
    fields:
      category:
        text: 1
      title:
        selector: td:nth-child(2)
      download:
        selector: a[href^="details.php?id="]
        attribute: href
        filters:
          - name: replace
            args: ["details.php?id=", "download.php?id="]
      details:
        selector: a[href^="details.php?id="]
        attribute: href
      grabs:
        selector: td:nth-child(7)
      files:
        selector: td:nth-child(3)
      size:
        selector: td:nth-child(6)
      seeders:
        selector: td:nth-child(8)
      leechers:
        selector: td:nth-child(9)
      date:
        selector: td:nth-child(5)
      downloadvolumefactor:
        case:
          "img.title:contains('Free Torrent!')": "0"
          "*": "1"
      uploadvolumefactor:
        case:
          "*": "1"
```

Waffles html:
http://pastebin.com/jQpG9HES

**Comment by RoloSoze** [comment]
@adamwinn This question is better directed to @kaso17 since he's the one who usually adds new trackers. I mostly identify & report bugs and enhancement opportunities, and nominate new trackers to be added. 

@kaso17 Are you able to help?

**Comment by adamwinn** [comment]
I finally figured this out.  It's fixed with #1205
